recipes/daemoniker/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "daemoniker" %}
+{% set version = "0.2.3" %}
+{% set hash = "d9b62c8d75227b3da88c0ca22b82e19d88d32dade4ffa838427235e0cdd5e578" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - daemoniker
+
+about:
+  home: https://github.com/Muterra/py_daemoniker
+  license: Unlicense
+  license_family: PUBLIC-DOMAIN
+  license_file: UNLICENSE.txt
```

I believe the license here is not correct, could you please double check?
also the license is being packaged on pypi so you do not need the license file
```suggestion
  license: LGPL-2.1
  license_family: LGPL
  license_file: LICENSE.txt
```

---

the ``LGPL-2.1`` is the correct license for the version ``0.2.3``
on the main branch they changed, but we cannot distribute the ``0.2.3`` as the latest license I believe

---

when they release a new version you can change the license to unlicensed

---

You are right, thanks!